src/machines/java/com/enderio/machines/common/item/FluidTankItem.java

```diff
@@ -26,4 +58,32 @@ public BlockEntityWithoutLevelRenderer getCustomRenderer() {
             }
         });
     }
+
+    @Override
+    public void appendHoverText(ItemStack stack, @Nullable Level level, List<Component> components, TooltipFlag flag) {
+        super.appendHoverText(stack, level, components, flag);
+        getTankCap(stack).ifPresent(handler -> {
+            for (int i = 0; i < handler.getTanks(); i++) {
+                FluidStack fluidStack = handler.getFluidInTank(i);
+                Component postFix = fluidStack.isEmpty() ? Component.literal("")
+                        : Component.literal(" ").append(fluidStack.getDisplayName());
+                components.add(
+                        Component.literal(fluidStack.getAmount() + " / " + handler.getTankCapacity(i)).append(postFix));
+            }
+        });
+    }
+
+    @Nullable
+    @Override
+    public ICapabilityProvider initCapabilities(ItemStack stack, @Nullable CompoundTag nbt) {
+        if (this instanceof FluidTankItem) {
```

Comment:
Why this check? We know this is a FluidTankItem

Reply:
I just copied this from the `BucketIem` where they do the same check. It does seem like an odd check, so happy to remove.